### PR TITLE
Fixed column status

### PR DESCRIPTION
**Pull request by BigOHenry**
Fix for https://github.com/contributte/datagrid/issues/803

### CODE REVIEW

**Comment by paveljanda** [comment]
Actually, it works fine for me. What data source do you use?

**Comment by BigOHenry** [comment]
Just tried version 6.2.0 and problem still persist. Data source is `Dibi\Fluent`.

**Comment by BigOHenry** [comment]
Problem still persist in 6.2.x.

File: src/Column/ColumnStatus.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function getColumn(): string
 	public function getCurrentOption(Row $row): ?Option
 	{
 		foreach ($this->getOptions() as $option) {
-			if ($option->getValue() === $row->getValue($this->getColumn())) {
+			if ($option->getValue() == $row->getValue($this->getColumn())) {
```
Comment (mabar) [inline src/Column/ColumnStatus.php]: Check for exact "empty" values would be better than ==

Comment (BigOHenry) [reply src/Column/ColumnStatus.php]: I made new PR because i deleted that old fork. https://github.com/contributte/datagrid/pull/827

**Pull request closed without merging**